cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/ec2metadata"
+	//"github.com/aws/aws-sdk-go/aws/endpoints"
```

Comment:
Please don't introduce commented out imports and code unless there's a very good reason for it (state it in comment if this is the case).